### ISSUE TITLE
Air Pressure

### ISSUE THREAD

**Issue by NogginTheNog** [naofireblade/homebridge-weather-plus#98]
tresholdAirPressure (compatibility "home" or "both")
At what treshold should the air pressure sensor trigger?

Shouldn't this be a trigger if it falls under the value, i.e there is a storm coming?

**Comment by naofireblade** [comment]
Good idea I will implement this in the future (low prio). Meanwhile if you use this in automations you can check for the inactive state of the sensor.

**Comment by NogginTheNog** [comment]
Actually a range would be handy as well

**Comment by dacarson** [comment]
Is a range only needed for Air Pressure?
Should the sensor trigger if it goes below a threshold or above a threshold? As noted it presently only triggers when it goes above the defined threshold.
